metals/src/main/scala/scala/meta/internal/rename/RenameProvider.scala

```diff
@@ -324,14 +323,14 @@ final class RenameProvider(
   ): MessageParams = {
     val renamed = name.map(n => s"to $n").getOrElse("")
     val message =
-      s"""|Cannot rename $old $renamed since it will change the semantics and 
+      s"""|Cannot rename $old $renamed since it will change the semantics and
```

Comment:
Double `and`

Reply:
thanks :) I hadn't actually changed those lines in this PR, but I'll take the chance to fix it